libraries/botbuilder-applicationinsights/tests/telemetryInitializer.test.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+
+const assert = require('assert');
+const appInsights = require("applicationinsights");
+const { TestAdapter, ActivityTypes, TelemetryLoggerMiddleware } = require('botbuilder-core');
+const { TelemetryInitializerMiddleware } = require('../');
+
+describe(`TelemetryMiddleware`, function () {
+    this.timeout(5000);
+
+    it(`telemetry initializer stores activity`, function (done) {      
```

Comment:
>telemetry initializer stores activity [](start = 8, length = 37)

May be valuable to do e2e with logging to make sure bot's session/userid actually land in the customEvent/dependency/trace/etc.

Reply:
@daveta I was thinking that the responsibility of this component is to 1. store the activity on the context and 2. optionally run the logging middleware.  Therefore that is what the tests for this particular class were going to cover.  I agree though that it might be worth adding tests to the class that actually grabs the activity and add the correlation properties.

Reply:
Seems like adding some additional asserts in the `.assertReply` might be the easiest way to validate.